### DISCUSSION TITLE
Received power in Receiver in Ray Tracing

### DISCUSSION THREAD

**Discussion by minrhmn** [NVlabs/sionna#431] [Q&A] [answered]
Hi! I was wondering if I can see/measure the received power in rx for a indoor ray tracing simulation?

**Comment by faycalaa** [comment, marked as answer]
Hi,

In general, you can compute the path loss from the channel impulse response (CIR) calculated by the ray tracer. The path loss represents the attenuation applied to the transmitted signal during its propagation. To estimate the received power, you need to scale the path loss obtained from the CIR by the transmit power.

To compute the path loss from the [path coefficients](https://nvlabs.github.io/sionna/api/rt.html#sionna.rt.Paths.a), you can sum their squared amplitudes over the path dimension.
You could also consider computing a [coverage map](https://nvlabs.github.io/sionna/api/rt.html#coverage-map).

You could therefore load a scene modelling an indoor scenario and compute the receive power. However, keep in mind that Sionna does not currently implement transmission (refraction), which is usually significant for indoor scenarios.

**Reply by potatoess11** [reply]
hi，I want to get the path loss of glass penetration in outdoor to indoor scenes, which is also the penetration loss, is there any good way

**Reply by jhoydis** [reply]
Sionna 1.0 supports now refraction.
